src/content/topics/home/organizing-flags/dashboard.mdx

```diff
@@ -17,6 +17,16 @@ Here is an image of the flag dashboard:
 
 ![The flag dashboard.](flags-dashboard.png)
 
+<Callout intent="info">
+<CalloutTitle>Flag dashboard features vary by plan</CalloutTitle>
+<CalloutDescription>
+
+  To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
```

Comment:
This is a slight rewording of the enterprise callout at the top of the code-references/index.mdx file